src/metricscollector/appmetricprune/prune/prune_test.go

```diff
@@ -0,0 +1,56 @@
+package prune_test
+
+import (
+	"time"
+
+	. "metricscollector/appmetricprune/prune"
+	"metricscollector/fakes"
+
+	"code.cloudfoundry.org/lager"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Prune", func() {
+	var (
+		metricDB        *fakes.FakeMetricsDB
+		logger          lager.Logger
+		prune           *Prune
+		timeBeforePrune time.Time
+		timeAfterPrune  time.Time
+		cutoffDays      int
+	)
+
+	BeforeEach(func() {
+
+		logger = lager.NewLogger("prune-test")
```

Comment:
Why does this need to be exposed? its only used when we create the Pruner below.